src/core/system.cpp

```diff
@@ -2191,7 +2191,7 @@ void System::UpdateSpeedLimiterState()
   float target_speed = m_turbo_enabled ?
                          g_settings.turbo_speed :
                          (m_fast_forward_enabled ? g_settings.fast_forward_speed : g_settings.emulation_speed);
-  m_throttler_enabled = (target_speed != 0.0f);
+  m_throttler_enabled = (target_speed != 1.0f);
```

Comment:
This logic doesn't sound correct. 1.0 is "normal" speed, where you want to throttle.

Reply:
Oh I see.. 

Then I think the m_throttler_enabled related interdependency between UpdateSpeedLimiterState() and ShouldUseVSync() should be removed.
(m_throttler_enabled will be finally set as 'false' at the end of  UpdateSpeedLimiterState() in case target_speed == 1.0f and ShouldUseVSync() return true.
But currently ShouldUseVSync() returns false in case m_throttler_enabled is 'true' and it is running at normal speed. So at the timing UpdateSpeedLimiterState() calls ShouldUseVSync(), it always return false and VSync will be never enabled.)

I think there's no need to check m_throttler_enabled in ShouldUseVSync() so I will simply remove it.